### PR TITLE
update banner to point to 2.16 post

### PR DESCRIPTION
**Pull request by wmorgan**

### CODE REVIEW

**Review by travisbeckham — COMMENTED** [review]
Did you want to update the copy to mention the new features, instead of the 2.15 features?

**Review by kflynn — CHANGES_REQUESTED** [review]
This is still talking about the Linkerd 2.15 features – we should probably talk about the 2.16 features. 🙂

File: linkerd.io/data/announcement_banner.yaml [hunk 1]
```diff
@@ -1,2 +1,2 @@
-description: "[Feb 21, 2024: Announcing Linkerd 2.15 with support for VM workloads, native sidecars, and SPIFFE! Read more &raquo;](/2024/02/21/announcing-linkerd-2.15/)"
+description: "[Aug 13, 2024: Announcing Linkerd 2.16 with support for VM workloads, native sidecars, and SPIFFE! Read more &raquo;](/2024/08/13/announcing-linkerd-2.16/)"
```
Comment (kflynn) [inline linkerd.io/data/announcement_banner.yaml:1]:
```suggestion
description: "[Aug 13, 2024: Announcing Linkerd 2.16 with Gateway API metrics, IPv6, policy auditing and more! Read more &raquo;](/2024/08/13/announcing-linkerd-2.16/)"
```

??

**Comment by wmorgan** [comment]
Oops, thank you both

**Review by travisbeckham — APPROVED** [review]

**Review by kflynn — APPROVED** [review]
Ship it! 🙂

**Pull request merged**